src/main/java/trade/wayruha/mexc/ws/WebSocketClient.java

```diff
@@ -88,10 +106,18 @@ public void subscribe(Collection<String> newChannels) {
             throw new IllegalArgumentException("Can't subscribe to more then " + WEB_SOCKET_MAX_CHANNELS_PER_CONNECTION + "channels in single WS connection");
         final Message msg = new Message(Action.SUBSCRIPTION, newChannels);
         final String content = objectMapper.writeValueAsString(msg);
-        send(content);
+        this.send(content);
         this.channels.addAll(newChannels);
     }
 
+    @SneakyThrows
+    public void ping(Collection<String> channels) {
+        final Message msg = new Message(Action.PING, channels);
+        final String content = objectMapper.writeValueAsString(msg);
+        this.send(content);
+        log.info("{} ping was send to channels {} ", logPrefix, channels);
```

Comment:
let's have log.debug() or even log.trace() here since info level would pollute the logs and the clients would have to disable INFO-logs for the lib (and lose some additional information). I would prefer TRACE level